01-creating-a-transaction/index.js

```diff
@@ -16,6 +16,11 @@ async function main() {
     const rows = await usersTable.getByPrimary('teamgreymass')
     const amount = rows[0].getColumn('amount')
 
+    // Or you can load the entries with a single method call
```

Comment:
This is unnecessary

Reply:
Yeah that's probably a good approach 👍

Reply:
We'll address this in the Contract Kit rounds.